Assets/Scripts/Gamestate/MatchController.cs

```diff
@@ -167,11 +170,17 @@ public void StartNextBidding()
         PlayerInputManagerController.Singleton.ChangeInputMaps("Bidding");
         MusicTrackManager.Singleton.SwitchTo(MusicType.BIDDING);
         onBiddingStart?.Invoke();
-        // TODO: Add Destroy on match win
+        // TODO: Add Destroy on match win   
+        //playerInputManagerController.RemoveListeners();
+        StartCoroutine(ShowLoadingScreen());
+    }   
+    private IEnumerator ShowLoadingScreen(){
+        loadingScreen.SetActive(true);
+        yield return new WaitForSeconds(6);
```

Comment:
The 6 here should be configurable, and perhaps sit on the loading screen class itself.